java/pom.xml

```diff
@@ -37,6 +37,17 @@
   <name>Fury Project Parent POM</name>
   <description>
     Apache Fury™ is a blazing fast multi-language serialization framework powered by jit and zero-copy.
+
+    Apache Fury (incubating) is an effort undergoing incubation at the Apache
```

Comment:
```suggestion
    Fury is an effort undergoing incubation at the Apache
```

Already "Apache Fury™" above, let's keep a consistent tidy format.

Reply:
Oh. It's about Javadoc footer? Than let's keep it as is.

Reply:
It's not about Javadoc footer, I will submit a PR later to fix this